### DISCUSSION TITLE
Capture and Insert scene

### DISCUSSION THREAD

**Discussion by Rudement** [SynthstromAudible/DelugeFirmware#1507] [Ideas]
I tried to explain this before, hopefully I'll do a better job this time. Looking for a "Capture & Insert Scene" function as Ableton has.  If you launch clips from different Grid Mode scenes (rows), you could capture all Clips (from different rows) you have playing and populate a new row (scene) with the clips you were playing. 
Here's how it works in Ableton.
https://www.youtube.com/watch?v=SbabP_kYIH8&t=198s

**Comment by lobit12** [comment]
ah.. ok good one.  actually I suggested something like this as well,  but it was more of an actual scene layer idea which would possibly be too complex of a way to get to the same place, which is to keep your clips more organized. 

another thing which came up in that discussion which is related to this would be a simple way to swap the position of 2 clips in a row and there was some talk of having it be similar to swapping kit elements ie,  hold down the two to be swapped and press a button.
